index.html

```diff
@@ -9,36 +9,214 @@
+        
+        <h3>TONIC</h3>
+
+        <div class="sub-title">
+          <h4>Canopy</h4>
+          <h4><i class="fas fa-circle"></i> Back End Dev</h4>
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <section>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </section>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+        <button class="see">See Project</button>
+      </ul>
+
+      <li class="work two">
+         <img src="./snapshot-portfolio2.png" alt="project-image2">
+         
+         <h3>MULTI-POST<br>STORIES</h3>
+
+        <div class="sub-title">
+          <h4>Canopy</h4>
+          <h4><i class="fas fa-circle"></i> Back End Dev</h4>
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <article>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </article>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+
+        <button class="see">See Project</button>
+
+
+      </li>
+
+
+      <li class="work three">
+        <img src="./snapshot-portfolio3.png" alt="project-image3">
+         
+         <h3>TONIC</h3>
+
+        <div class="sub-title">
+          <h4>Canopy</h4>
+          <h4><i class="fas fa-circle"></i> Back End Dev</h4>
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <article>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </article>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+
+        <button class="see">See Project</button>
+
+      </li>
+
+      <li class="work four">
+        <img src="./snapshot-portfolio4.png" alt="project-image4">
+         
+         <h3>MULTI-POST <br>STORIES</h3>
+
+        <div class="sub-title">
+          <h4>Canopy</h4>
+          <h4><i class="fas fa-circle"></i> Back End Dev</h4>
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <article>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </article>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+         <button class="see">See Project</button>
+        
+        </li>
+
+      <li class="about">
```

Comment:
- [ ] The container of this section is a `<section>` tag, as it is a requirement and for semantic purposes as explained above. 
- [ ] The content should be aligned with flexbox. Please kindly fix that.